shapers/base.lua

```diff
@@ -67,8 +67,18 @@ function shaper:measureChar (char)
    local options = SILE.font.loadDefaults({})
    options.tracking = SILE.settings:get("shaper.tracking")
    local items = self:shapeToken(char, options)
-   if #items > 0 then
-      return { height = items[1].height, width = items[1].width, depth = items[1].depth }
+   SU.dump({ char, items })
+   if #items >= 1 then
+      local item = items[1]
+      if item.name == ".notdef" then
```

Comment:
Would checking item.gid be more robust than a name check? Normally, if I read that well, .notdef is supposed to be GID 0 according to [Microsoft Typography](https://learn.microsoft.com/en-us/typography/opentype/spec/recom#glyph-0-the-notdef-glyph)
I am no font expert -- my initial suggestion in the original issue was checking the GID 0, and also for "additional safeguard", the .notdef name, and the .null name -- I am not sure regarding the latter and how I sort of guessed it there (from what I read now it seems the NULL / .null was something from Apple's TTF, but no longer applies to OTF -- Perhaps I still have some old TTF's ? Dunno).

Reply:
Probably. I am not sure there is a 100% solution with one check, we probably need to combine a bunch of possible combinations of results. he thing is don't have the energy to ferret them all out right now and I'm willing to start with something. Adding checks that cover more possible failure modes should be easy as long as we do it with some boolean return value.

I did just change the default check to checking the GID for 0.